### PR TITLE
chore: update wot-typescript-definition

### PR DESCRIPTION
**Pull request by erossignon**
update to latest "wot-typescript-definition" ( version 21) 
and use "pinned" versions to avoid subtle side effects when minor breaking changes are introduced in a patched version.

### CODE REVIEW

**Review by relu91 — APPROVED** [review]
LGTM, package-lock should avoid these "side-effects" for minor releases, but it does not hurt to pin the version.

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
         "check:versions": "node utils/check_package_version_consistency.js",
         "build:docker": "docker build -t wot-servient .",
         "clean:dist": "npm exec  --workspaces -- npx rimraf tsconfig.tsbuildinfo dist",
+        "update:wot-typescript-definitions": "npx npm-check-updates  -u -f \"wot-typescript-definitions\" --deep",
```
Comment (danielpeintner) [inline package.json:34]: I wonder whether we want to have that. @erossignon you added a similar check for opcua in another PR.
@relu91 do you think this is fine?

Comment (erossignon) [reply package.json:34]: I find this very useful to consistently upgrade a particular module used everywhere in a mono repo.

Comment (danielpeintner) [reply package.json:34]: I see your arguments

**Review by danielpeintner — APPROVED** [review]

**Pull request merged**